include/net_manager.hpp

```diff
@@ -170,12 +171,6 @@ class NetManager {
         if (it == message.end()) {
           return;
         }
-        auto exceptionMessage = it->get<std::string>();
-        it = message.find("code");
-        if (it == message.end()) {
-          return;
-        }
-        auto code = it->get<std::string>();
```

Comment:
this caused an unhandled exception since code is a number being parsed as string, since we don't use these valaues removed them